app/src/main/java/ch/epfl/sweng/SDP/game/drawing/items/BumpingItem.java

```diff
@@ -64,6 +61,11 @@ public void activate(final PaintView paintView) {
                         * (this.getRadius() + paintView.getCircleRadius() + 5)));
     }
 
+    @Override
+    public void deactivate(PaintView paintView) {
+        // Must be implemented, but does nothing.
+    }
+
```

Comment:
You could pull up this method so that you don’t have to override twice (here and in AddStarsItem)

Reply:
Pull up?

Reply:
Slide 53 of the refactoring lesson. In other words replace the abstract deactivate method of the parent by this default method.